Create Modeling and Desc

huyanh0720 opened the pull request:
- Create Data Modeling with 7 Dim Tables: insurance, demographic, patient, date, Diagnosis, symptom, disease, and 1 Fact Table: Medical
- Create a CSV file that includes Desc for each table and Example tables.

huyanh0720 committed:
Add files via upload

- Create Data Modeling with 7 Dim Tables: insurance, demographic, patient, date, Diagnosis, symptom, disease, and 1 Fact Table: Medical
- Create a CSV file that includes Desc for each table and Example tables.

duy-deliver reviewed (commented):
`Delete Demographic : Because all attribute at Demographic all ready have in Insurance and Patient`
`Move history_covid from Demographic to Patient`
`Delete DateID from Medical`
`Move Appointment_Date from Medical to Diagnosis`

huyanh0720 committed:
New Data Model and Desc, Example file

- Added new Desc and Example Tables CSV file and New Data Model

duy-deliver reviewed (commented):
```
- Change Insurance_Expired from Insurance table to random (year =2024)
- Change Appointment_Date from Diagnosis table to random(year =2021,2022)
- Remove Address from Patient
- Diagnosis: Adding Appointment_date
```
- Medical: 
```
+ Remove Date_ID, Insurance_Number
+ Adding History_Covid, Appointment_Date
```

huyanh0720 committed:
Update Desc file and Model as requested.

- Change Insurance_Expired from Insurance table to random (year =2024)
- Change Appointment_Date from Diagnosis table to random(year =2021,2022)
- Delete Address from Patient table
-Put all the IDs on the top from each Tables
- Add Appointment_Date from Diagnosis table
- Delete Date_ID From Medical and Insurance_Number
- Add History_Covid and Appointment_Date for Medical Table
- Add Blood_Pressure_mmHg and Blood_Sugar_mgDl to Patient table

duy-deliver reviewed (commented):
`Appointment_Date in Diagnosis must be between ( 2019 - 2021 ). `

huyanh0720 committed:
Update Desc File

- Fix the Desc of Apointment_Date range to (2019-2021)

duy-deliver reviewed (approved):
Ok , approved
